stored_requests/config/config.go

```diff
@@ -201,6 +194,9 @@ func newFetcher(cfg *config.StoredRequestsSlim, client *http.Client, db *sql.DB)
 
 	if cfg.Files.Enabled {
 		fFetcher := newFilesystem(cfg.Files.Path)
+		if !cfg.Files.LazyLoad {
+			fFetcher.(stored_requests.CategoryFetcher).FetchAllCategories(context.Background())
+		}
```

Comment:
I think this should live elsewhere so this function always just returns a fetcher. Perhaps this should live in the `newEventProducers` function and we use an event to perform the initial load like the other backends are. This logic can probably be combined with the logic run during polling which doesn't exist for file fetching right now but we should probably add.

Reply:
The categories fetcher wants to store the map in itself so creating another fetcher like `fileEvents.NewFilesLoader` does is a no-go, it was hard to find another place that has both the fetcher and its external config. This should be revisited if categories fetcher is refactored.

Reply:
It's non-obvious through the code that this is prefetching the files to keep in a memory store. Perhaps there's a better way of expressing this, or else consider adding a comment.

Reply:
I agree that it's not obvious, I will add a comment.